src/components/dcatd-form.scss

```diff
@@ -0,0 +1,242 @@
+@import "../styling/config.scss";
+
+.ui {
+  &.modal {
+    bottom: auto !important;
+  }
+}
+
+.container-main {
+  margin: 75px 0;
+}
+
+.dcatd-form {
+  .control-label {
+    @extend %heavy-font;
+
+    font-size: 16px;
+    margin-bottom: 2px;
+  }
+
+  .field-description {
+    @extend %light-font;
+
+    color: $brownish-grey;
+    font-size: 14px;
+  }
+
+  .form-control {
+    @extend %base-font;
+
+    border: 1px solid $brownish-grey !important;
```

Comment:
because the styling in MVP has lowest priority and I had to suppress some react semantic ui styling. when we go forward next time we will refactor this.